File: doc/codes/python/examples/api/compute/modulation/radiation.py
from autd3.units import Hz
from autd3_modulation import SineOption, modulation_buffer, radiation_pressure, sine

src = modulation_buffer()
sine(150.0 * Hz, SineOption(), src)

dst = modulation_buffer()
# ANCHOR: api
radiation_pressure(src, dst)
# ANCHOR_END: api

File: doc/codes/csharp/examples/api/compute/modulation/radiation.cs
using AUTD3;
using static AUTD3.Units;

namespace DocSamples.ApiComputeModulationRadiation;

internal static class Sample
{
    internal static void Run()
    {
        var src = Modulation.ModulationBuffer();
        Modulation.Sine(150 * Hz, new SineOption(), src);

        var dst = Modulation.ModulationBuffer();
        // ANCHOR: api
        Modulation.RadiationPressure(src, dst);
        // ANCHOR_END: api
    }
}
